lib/data_hygiene/bulk_organisation_updater.rb

```diff
@@ -35,10 +35,14 @@ def process_row(row)
 
     def find_document(row)
       slug = row.fetch("Slug")
+
+      return if slug.nil?
+      return if slug.strip.empty?
```

Comment:
Since `"  ".blank?` is `true`, we can do
```suggestion
      return if slug.blank?
```

Reply:
Sure, do you want to make the change? I don't know how to use the "suggest change" thing on GitHub.

Reply:
If I make the change then I can't approve the PR. If the change looks good to you, you can just edit the commit and force push.

Reply:
Ok, I've incorporated this in to the first commit now.